tests/test_image.py

```diff
@@ -1,11 +1,11 @@
 from music21 import *
 
 def test_png_lilypond():
-    a = converter.parse('tests/sample.musicxml').write('lilypond.png')
+    a = converter.parse('tests/testFiles/Chant.musicxml').write('lilypond.png')
 def test_png_mscore_from_mxl_short():   
-    easy_task = converter.parse('tests/sample.musicxml').write('musicxml.png')
+    easy_task = converter.parse('tests/testFiles/Chant.musicxml').write('musicxml.png')
```

Comment:
Can we use a different png file name? This case and below are using the same one "musicxml.png"

Reply:
`musicxml.png` is used to indicate music21 to convert to png using musescore, the file name always ends with `.png` only.